apps/emqx/test/emqx_static_checks.erl

```diff
@@ -19,10 +19,12 @@
 -compile(export_all).
 -compile(nowarn_export_all).
 
--include_lib("emqx/include/logger.hrl").
 -include_lib("common_test/include/ct.hrl").
 -include_lib("stdlib/include/assert.hrl").
 
+-define(NOTICE(Fmt, Args), io:format(user, "[NOTICE] " ++ Fmt, Args)).
```

Comment:
I know it's only in this file and the one above, but is there a reason to not include these definitions in the logger.hrl file and keep importing that file rather than defining these macros in more than one location? Is it because these are test files? Would it be worth creating a test helper/logger file and define these once and import that?

Reply:
there is actually a ?ULOG macro which prints to `user` device.
will make use of that.